Lib/getpass.py

```diff
@@ -126,6 +146,13 @@ def fallback_getpass(prompt='Password: ', stream=None):
     return _raw_input(prompt, stream)
 
 
+def _is_ascii(echochar):
+    # ASCII excluding control characters
+    if echochar and (32 <= ord(echochar) <= 127):
```

Comment:
```suggestion
    if echochar is None or (32 <= ord(echochar) <= 127):
```

Reply:
`echochar` is None by default. why?

Reply:
It should work only if we pass `echochar` arg